exchanges/sharedtestvalues/sharedtestvalues.go

```diff
@@ -40,7 +40,7 @@ func NewTestWebsocket() *stream.Websocket {
 		Init:              true,
 		DataHandler:       make(chan interface{}, 75),
 		ToRoutine:         make(chan interface{}, 1000),
-		TrafficAlert:      make(chan struct{}),
+		TrafficAlert:      make(chan string),
```

Comment:
I think in future iterations we can instead pass a pointer to websocketconnection ( stream.Connection interface ) as this flushes both connections at the same time when it might not be needed.

Reply:
Yes, it's a good idea, to pass pointer instead values.